File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/sys_Control_DefaufValueService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class sys_Control_DefaufValueService : BaseService<sys_Control_DefaufValue, Isys_Control_DefaufValueRepository>
    , Isys_Control_DefaufValueService
    {
    private readonly Isys_Control_DefaufValueRepository _sys_Control_DefaufValueRepository;
    public sys_Control_DefaufValueService(Isys_Control_DefaufValueRepository sys_Control_DefaufValueRepository) : base(sys_Control_DefaufValueRepository)
    {
    _sys_Control_DefaufValueRepository = sys_Control_DefaufValueRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/sys_Control_DefaufValue.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { sys_Control_DefaufValue } from './sys_Control_DefaufValue.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class sys_Control_DefaufValueService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Control_DefaufValue_Id', 'Funtions_Class', 'Control_Name', 'DefaufValue', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_id'];

    List: sys_Control_DefaufValue[] | undefined;
    ListFilter: sys_Control_DefaufValue[] | undefined;
    FormData!: sys_Control_DefaufValue;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "sys_Control_DefaufValue";
    }
}

